.storybook/preview-head.html

```diff
@@ -8,4 +8,9 @@
     body {
         font-family: "Lato", sans-serif;
     }
+
+    /* Overrides the ArgTypes table tabs to avoid some issues with Popper */
```

Comment:
Would be good to clarify what issues so that we can know if we can remove this safely in the future.